smartsim/_core/launcher/colocated.py

```diff
@@ -131,16 +129,11 @@ def _build_colocated_wrapper_cmd(
     fs_cmd = []
     if custom_pinning:
         fs_cmd.extend(["taskset", "-c", custom_pinning])
+    # TODO add a new backend
     fs_cmd.extend(
-        [CONFIG.database_exe, CONFIG.database_conf, "--loadmodule", CONFIG.redisai]
+        [CONFIG.database_exe, CONFIG.database_conf]
```

Comment:
In theory, none of these attrs will make sense in the SSv1.0 API. If you can get it by the CI, consider dropping these properties from the `Config` class

Reply:
removed